cmd/utils/flags.go

```diff
@@ -757,14 +762,23 @@ func SetupNetwork(ctx *cli.Context) {
 		NetworkIdFlag.Value = 0
 		core.BlockReward = big.NewInt(1.5e+18)
 		core.ExpDiffPeriod = big.NewInt(math.MaxInt64)
+	default:
+		params.TargetGasLimit = common.String2Big(ctx.GlobalString(TargetGasLimitFlag.Name))
```

Comment:
True I guess. And on a side note, I think we should remove olympic at some point.